iocAdmin/src/iocReleaseCreateDb.py

```diff
@@ -47,19 +47,25 @@ def export_db_file( module_versions, path=None):
         x = key.replace("_MODULE_VERSION","",1)
         if idx >= idxMax:
             break
-        print('record(stringin, "$(IOC):RELEASE%02d") {' % idx, file=out_file)
+        print('record(%s, "$(IOC):RELEASE%02d") {' % ('lsi' if is_epics7 else 'stringin', idx), file=out_file)
+        if is_epics7:
+            print('  field(INP, {const:"%s"})' % module_version, file=out_file)
```

Comment:
Is there an advantage to adding the string to the INP field instead of directly to VAL? For the LSI record, does INP accept 65535 characters, too?